src/uglify/serialization.js

```diff
@@ -0,0 +1,46 @@
+const toType = value => (Object.prototype.toString.call(value).slice(8, -1));
```

Comment:
@mikesherov I refactored the serialization.js, and now the user can know the details of the error :-)
```
Error: options serialization failed: "test" parse failed: 'console' is not defined.
    1| function toBeSerialized() {
 >> 2|     console.log(a);
    3|   }
```